### PR TITLE
cmd/info: add '--recursive' flag to get summary of a tree

### PR DESCRIPTION
**Pull request by SandyXSD**

### CODE REVIEW

File: cmd/info.go [hunk 1]
```diff
@@ -79,10 +88,11 @@ func info(ctx *cli.Context) error {
 			continue
 		}
 
-		wb := utils.NewBuffer(8 + 8)
+		wb := utils.NewBuffer(8 + 9)
```
Comment (davies) [inline cmd/info.go:91]: For mountpoint using older version, what will happen?

File: pkg/vfs/internal.go [hunk 2]
```diff
@@ -155,9 +155,10 @@ func handleInternalMsg(ctx Context, cmd uint32, r *utils.Buffer) []byte {
 	case meta.Info:
 		var summary meta.Summary
 		inode := Ino(r.Get64())
+		recursive := r.Get8()
```
Comment (davies) [inline pkg/vfs/internal.go:158]: IF the client is older version, there will be no extra byte .

File: cmd/info.go (hunk 1 above)
Comment (SandyXSD) [reply cmd/info.go:91]: The last byte will be ignored, as older clients will always return the recursive result

File: pkg/vfs/internal.go (hunk 2 above)
Comment (SandyXSD) [reply pkg/vfs/internal.go]: 459c3bc

**Pull request merged**